demos/TicTacToe/setup_demo.py

```diff
@@ -59,35 +59,40 @@ def main():
   game_name = "tic_tac_toe_demo"
   game_resources_dir = os.path.join(ROOT_DIRECTORY, "game_resources")
   game_files_dir = os.path.join(ROOT_DIRECTORY, game_name);
-  windows_proj_dir = os.path.join(game_files_dir,"proj.win32")
-  mac_proj_dir = os.path.join(game_files_dir,"proj.ios_mac","mac")
-  linux_proj_dir = os.path.join(game_files_dir,"proj.linux")
-  build_dir = os.path.join(game_files_dir,"build")
-  executable_dir = os.path.join(build_dir,"bin",game_name,"Debug")
+  windows_proj_dir = os.path.join(game_files_dir, "proj.win32")
+  mac_proj_dir = os.path.join(game_files_dir, "proj.ios_mac","mac")
+  linux_proj_dir = os.path.join(game_files_dir, "proj.linux")
+  build_dir = os.path.join(game_files_dir, "build")
+  executable_dir = os.path.join(build_dir, "bin", game_name, "Debug")
+
+  # Checks whether the google-services.json exists in the debug directory.
+  if not os.path.isfile(os.path.join(ROOT_DIRECTORY, "google_services", "google-services.json")):
+    # Runs the tic-tac-toe executable.
+    logger.error("google_services/google-services.json is missing.")
+    exit()
 
   # Creating the cocos2d-x project. 
-  log_run(ROOT_DIRECTORY, logger,"cocos new {0} -p com.DemoApp.{0} -l cpp -d .".format(game_name))
+  log_run(ROOT_DIRECTORY, logger, "cocos new {0} -p com.DemoApp.{0} -l cpp -d .".format(game_name))
 
   # Removing the default cocos2d-x project files.
-  log_run(ROOT_DIRECTORY, logger,"rm -r {0}/Classes {0}/Resources {0}/CMakeLists.txt".format(game_files_dir) )
+  log_run(ROOT_DIRECTORY, logger, "rm -r {0}/Classes {0}/Resources {0}/CMakeLists.txt".format(game_files_dir))
 
   # Copies the google-services.json file into the correct directory to run the executable. 
-  log_run(ROOT_DIRECTORY, logger,"cp google_services/google-services.json {}".format(os.path.join(game_resources_dir,"build","bin", game_name, "Debug")))
+  log_run(ROOT_DIRECTORY, logger, "cp google_services/google-services.json {}".format(os.path.join(game_resources_dir, "build", "bin", game_name, "Debug")))
```

Comment:
Can these be split up unto multiple lines? They surely must exceed the 80 character limit, no?

Reply:
I went ahead and installed the linting and formatting suggested in the https://google.github.io/styleguide/pyguide.html. So the following commit will address this and all of the other formatting issues that are present in this iteration.